packages/titus-backend/.env.sample

```diff
@@ -45,6 +45,10 @@ SECRETS_PG_PASS=PG_PASS
 #
 # SECRETS_STRATEGY=aws
 # SECRETS_PG_PASS=DB_PASSWORD
-
+#
+# This is an example for Azure Key Vault
+#
+# SECRETS_STRATEGY=azr
```

Comment:
let's name the strategy azure for consistency

Reply:
Will do! (btw I used 3 chars for consistency 😅)

Reply:
fair enough, a different definition of consistency. let's stick to the "official" names: gcp, aws, azure